doc/source/conduce-entities.rst

```diff
@@ -0,0 +1,159 @@
+.. _conduce-entities:
+
+================
+Conduce Entities
+================
+
+.. note:: WIP
+
+Conduce uses a concept called entity to define objects that exist in space and time.  An entity can exist for an instant or across a time span.  It can describe a point in space, a path (sequence of points), or a region (polygon).  An entity is defined by a series of samples that describe it's location in space over a period of time.  An entity also has attributes that describe it's characteristics. An entity's attributes can change over time.
+
+-----------------
+Creating entities
+-----------------
+
+An entity is a series of samples that describe the state of and object over a period of time.  A sample is the representation of an entity's momentary state.  Samples are created from user data.  The data should describe people or things that exist in some location during some time span.  Data can be ingested from static files with thousands of records or real-time streams that provide state updates in real-time.  As an example we'll ingest the following real-time shipment data into an existing dataset named ``shipments``:
+
+.. list-table:: Source data: real-time shipment status
+   :header-rows: 1
+   :widths: auto
+
+   * - ID
+     - Method
+     - Date
+     - Latitude
+     - Longitude
+     - Value
+     - Status 
+   * - 1
+     - ground
+     - 2017-10-24T10:22:09+00:00
+     - 37.952861
+     - -91.922607
+     - $102,325.26
+     - transit
+   * - 2
+     - air
+     - 2017-10-24T10:22:11+00:00
+     - 38.289937
+     - -100.107422
+     - $3,216,757.84
+     - transit
+   * - 3
+     - ground
+     - 2017-10-24T10:22:08+00:00
+     - 40.354917
+     - -89.104614
+     - $204,694.41
+     - processing
+   * - 4
+     - air
+     - 2017-10-24T10:22:06+00:00
+     - 35.85344
+     - -115.993652
+     - $102,325.26
+     - transit
+   * - 5
+     - freight
+     - 2017-10-24T10:22:18+00:00
+     - 25.029996
+     - -134.318848
+     - $1,431,873,345.01
+     - transit
+
+The data in this table represents the present location of 5 shipments.  Each shipment has an ID, a shipping method, a date when it was last updated and a location in geographic coordinates.  Additionally the current state and value of the shipment is documented.  A Conduce entity is able to capture and represent all of this information.  These data records will be ingested as the first samples of 5 unique entities of the ``shipments`` dataset.
+
+An entity sample requires four fields:
+ **id**
+     a unique string that represents a specific object that exists.
```

Comment:
In general, I think it should be clearer that an entity = ID.  So all the samples for a given entity point to the same ID.

Right here, I'd say "a unique string that represents a specific ~object~ entity that exists".